package.json

```diff
@@ -2,19 +2,20 @@
   "name": "eslint-find-rules",
   "version": "0.0.0-semantically-released",
   "description": "Find built-in ESLint rules you don't have in your custom config.",
-  "main": "src/lib/rule-finder.js",
+  "main": "dist/lib/rule-finder.js",
   "scripts": {
     "cover": "nyc --reporter=lcov --reporter=text npm test",
-    "test": "mocha --recursive",
-    "lint": "eslint --ignore-pattern test/fixtures .",
+    "test": "xo && mocha --recursive",
     "update-contributors": "all-contributors generate",
     "commit": "git-cz",
     "validate": "npm-run-all --parallel lint cover --sequential check-coverage",
```

Comment:
`lint` should be either replaced with `test` or the separation of the `test` and `lint` npm scripts should be kept.
I'd prefer the latter.